var/spack/repos/builtin/packages/wget/package.py

```diff
@@ -55,6 +56,7 @@ def configure_args(self):
 
         args = [
             '--with-ssl={0}'.format(spec.variants['ssl'].value),
+            '--without-included-regex',
```

Comment:
Does this option apply in general or only for macOS builds?  Do you want to add a comment for why the option is being specified?

Reply:
It's definitely needed to build on macOS with Apple Clang, not sure what else it might affect. From the help message:
```
  --without-included-regex
                          don't compile regex; this is the default on systems
                          with recent-enough versions of the GNU C Library
                          (use with caution on other systems).
```

Reply:
Would it be prudent at this point then to only add the option for macOS builds?

Reply:
Up to you. It would definitely be good for someone to test this on Linux just to make sure it doesn't break things. But Linux usually comes with wget, so I'm not sure how many people are using Spack to install wget on any system other than macOS.

Reply:
Built fine for me on a Linux box.